tests-builds-from-more-prs/README.md

```diff
@@ -0,0 +1,76 @@
+# Triggering testing farm tests with builds from other PRs (projects)
+
+Use case: having 2 builds for different PRs, being able to trigger TF tests using both builds with a Packit
+command comment in one PR
```

Comment:
Yesterday, we discussed with Tomáš, if it makes sense to cover builds as well.

Hopefully, most use-cases can be covered by a shared Copr project and (re)triggering the builds in the correct order. So it's probably not needed but we should at least have this in mind. And also, this might be a prerequisite to running the tests with correct build versions.